init_test_db.sh

```diff
@@ -0,0 +1,6 @@
+psql -U postgres -c "create user aatams_test unencrypted password 'aatams_test';"
```

Comment:
Btw, one way to fix this would be to factor out the `-U postgres` bit in to an environment variable.